pkgs/tools/security/rng-tools/default.nix

```diff
@@ -41,7 +46,7 @@ stdenv.mkDerivation rec {
     ++ optionals withGcrypt        [ libgcrypt ]
     ++ optionals withJitterEntropy [ jitterentropy ]
     ++ optionals withNistBeacon    [ openssl curl libxml2 ]
-    ++ optionals withPkcs11        [ libp11 openssl ];
+    ++ optionals withPkcs11        [ libp11 opensc openssl ];
```

Comment:
Do you need it here? I pretty sure nix is smart enough to propagate opensc as a dependency just because the path is in the source file.
It doesn't actually use it as a regular library - it really just needs the path there and OpenSSL will load it on demand.
I guess it's up to taste whether to include it here.

Reply:
It's technically not needed, you're right, so let's get rid of it.